linux_os/guide/system/accounts/accounts-physical/require_emergency_target_auth/oval/shared.xml

```diff
@@ -5,9 +5,11 @@
     <criteria operator="AND">
       <criterion comment="Conditions are satisfied"
       test_ref="test_require_emergency_service" />
+      {{%- if product not in ["ol8", "rhel8"] -%}}
       <criterion test_ref="test_require_emergency_service_emergency_target" />
       <criterion test_ref="test_no_custom_emergency_target" negate="true"/>
```

Comment:
These changes were intended to comply more precisely with DISA STIG ids OL08-00-010152 and RHEL-08-010152. But from what you say apparently all the extra criteria help to verify that the system behaves as DISA expects. So would you say these changes aren't actually needed?

Reply:
Yes, I consider these extra tests important to make sure the requirement is achieved as expected. Otherwise it would be necessary to create new rules to assess these criteria and make sure all these related rules are selected during the assessment. However, for this specific case, I don't think it is reasonable to decouple these extra tests. The granularity is good when makes the rules simpler, but not easier to be fooled, as I mentioned with an example from my other comment.

Reply:
Ok, I'll close this PR then